src/templates/input.html

```diff
@@ -2,7 +2,7 @@
 {% from "_includes/forms" import time %}
 <tr>
     <th>
-        {{ craft.i18n.getLocaleData().getWeekDayName(day) }}
+        {{ craft.i18n.getLocaleData().getWeekDayName(day) | ucfirst }}
```

Comment:
Is there a place where this was necessary? Week day names should already be correctly capitalized.